Assets/Scripts/Strategy/GameResources/IResource.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SwordAndBored.Strategy.GameResources
+{
+    public interface IResource
+    {
+        //List<IResourceSubscriber> Subscribers { get; set; }
+        //The amount of resource you have
+        int Amount { get; set; }
+
+        /**
+         * returns true if the Purchase can be afforded
+         */
```

Comment:
Use the XML Document comments (`///`). Visual Studio should help you generate them